extensions/amp-story-page-attachment/0.1/amp-story-draggable-drawer.js

```diff
@@ -237,7 +237,9 @@ export class DraggableDrawer extends AMP.BaseElement {
       ? this.startListeningForTouchEvents_()
       : this.stopListeningForTouchEvents_();
 
-    this.headerEl.toggleAttribute('desktop', !isMobile);
+    isMobile
+      ? this.headerEl.removeAttribute('desktop')
+      : this.headerEl.setAttribute('desktop', '');
```

Comment:
See `toggleAttribute` in https://github.com/ampproject/amphtml/blob/bcf39f30a3b23fcb06ac25a875206a38567a3c45/src/core/dom/index.js#L457

Reply:
Awesome. Thank you Justin.